test/e2e/e2e_test.go

```diff
@@ -356,7 +399,8 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				for i := 0; i < numConcurrentPods; i++ {
 					pvcs := []*v1.PersistentVolumeClaim{}
 					for j := 0; j < volsPerPod; j++ {
-						pvc := e2epv.MakePersistentVolumeClaim(makeLocalPVCConfig(config, DirectoryLocalVolumeType), config.ns)
+						pvcConfig := makeLocalPVCConfig(config, DirectoryLocalVolumeType)
```

Comment:
what's the difference with before?

Reply:
no difference, probably I did something in between like logging and forgot to make it like it was before